### PR TITLE
Fixes #5

### LINKED ISSUE #5: Dockerize the set up.
Backend + Frontend needs to be dockerized replacing the current sqlite3 set up with Postgres. This is not an issue I would be picking up right now. I am leaving it up to the community to help out.

### PR DESCRIPTION
**Pull request by abheektripathy**
Created separate docker-files for frontend and backend.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+version: 3.7
```
Comment (0x0elliot) [inline docker-compose.yaml:1]: It should be a string.

Comment (0x0elliot) [reply docker-compose.yaml:1]: Change it to "3.7" instead of it just being a float.

Comment (abheektripathy) [reply docker-compose.yaml:1]: okay.

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+services:
+  django_gunicorn:
+    volumes:
+      - static:/static
+    env_file:
+      - .env
```
Comment (0x0elliot) [inline docker-compose.yaml:8]: Okay so, I know for a fact that we would be creating lots of .env files now. I can see the next being created for Postgres itself. Now, The best thing to do is to create a folder called "docker" in the main directory of the project, Move the two Dockerfiles you have created carefully to it. Make the changes required. Make sure that, 

The Backend Dockerfile is called --> Dockerfile_gunicorn
The Frontend Dockerfile is called --> Dockerfile_react

We will add nginx configs later. Right now, Get these changes done.

Then in the new `docker` file created I want you to lay down the structure for .env file in .env_file_app_template

The user would be supposed to copy this to env_file_app from env_file_app_template which your Dockerfile_gunicorn should read from.

Kind of like this here: https://github.com/intelowlproject/IntelOwl/blob/master/docker/env_file_app_template

In that case, They would have to move the file env_file_app_template to env_file_app. Mention this in the README.md for the time being.

Comment (0x0elliot) [reply docker-compose.yaml:8]: When i say a `docker` folder, I want it to look somewhat like this: https://github.com/intelowlproject/IntelOwl/tree/master/docker

Comment (0x0elliot) [reply docker-compose.yaml:8]: Later on, When we create a Postgres container as well, We would need them to create an env file for postgres as well. But to take smart measures, We would instead create env_file_postgres_template and state in README.md to move the template file to env_file_postgres.

Also make sure to add env_file_postgres and env_file_app to `.gitignore`. We don't want the users or the developers to push sensitive data anywhere by mistake.

Comment (0x0elliot) [reply docker-compose.yaml:8]: Also, change the name of `docker-compose.yaml` to `docker-compose.yml`. I just like it better that ways d:

Comment (0x0elliot) [reply docker-compose.yaml:8]: Also, In your docker-compose file, you assume that the user has built the images beforehand. don't do that. link the docker files for backend and frontend to their appropriate entries in the docker-compose file properly. Kind of like how I did here: https://github.com/ArgusOSS/Uturn-web/blob/a15acaba410f5e0d8c90717edbc3e851a5294fe9/docker-compose.yml#L12

Comment (abheektripathy) [reply docker-compose.yaml:8]: Will get this done by tonight.

Comment (0x0elliot) [reply docker-compose.yaml:8]: Take your time.

**Comment by 0x0elliot** [comment]
Since you are at it as well, I would recommend that you implement a Postgres container as well, add it to the same network as the Backend through the docker-compose file and connect it nicely with django. If you have any troubles here, We can get on a discord call to resolve it.

**Comment by 0x0elliot** [comment]
good progress. i am loving it. just change the sqlite3 support to postgres for the project. great efforts, thank you!

File: .gitignore [hunk 3]
```diff
@@ -6,4 +6,5 @@ db.*
 .env
```
Comment (0x0elliot) [inline .gitignore:6]: change the file name `.env` to env_file_app please.

File: Docker/env_file_postgres_template [hunk 4]
```diff
@@ -0,0 +1,3 @@
+POSTGRES_PASSWORD=password
+POSTGRES_USER=user
+POSTGRES_DB=intel_owl_db
```
Comment (0x0elliot) [inline Docker/env_file_postgres_template:3]: bahahaha. yes make this variable something Eye related

Comment (abheektripathy) [reply Docker/env_file_postgres_template:3]: F

File: Docker/docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,31 @@
+version: "3.7"
+
+services:
+  django_gunicorn:
+    volumes:
+      - static:/static
+    env_file:
```
Comment (0x0elliot) [inline Docker/docker-compose.yml:7]: two env_files d:?

File: Docker/Dockerfile_react [hunk 6]
```diff
@@ -0,0 +1,4 @@
+FROM node:17-alpine
+
```
Comment (0x0elliot) [inline Docker/Dockerfile_react:2]: install yarn/npm inside this docker container

File: Docker/Dockerfile_react [hunk 7]
```diff
@@ -0,0 +1,4 @@
...
+WORKDIR /app
+
```
Comment (0x0elliot) [inline Docker/Dockerfile_react:4]: make the frontend run after installing dependencies.

**Pull request closed without merging**